crypto/bls12377/arithmetic_decl.go

```diff
@@ -1,3 +1,4 @@
+//go:build amd64 && !generic
 // +build amd64,!generic
```

Comment:
Go 1.18 prefers the new format of build instructions and will add them during formatting. Once this has been added by gofmt 1.18, this output stays stable across all gofmt versions between 1.15 and 1.18.

Eventually, we can use `go fix` to remove the old instructions. But I want to wait until I am sure at least go 1.18 is used everywhere.